CorePayments/src/main/java/com/paypal/android/corepayments/apis/eligibility/FundingEligibilityIntent.kt

```diff
@@ -0,0 +1,5 @@
+package com.paypal.android.corepayments.apis.eligibility
+
+internal enum class FundingEligibilityIntent {
+    SALE, CAPTURE, AUTHORIZE, ORDER, TOKENIZE, SUBSCRIPTION
```

Comment:
Update to match the iOS PR:
```suggestion
    CAPTURE, AUTHORIZE
```